main.go

```diff
@@ -28,7 +28,7 @@ const (
 
 var (
 	version = "" // Must not be const, set using ldflags at build time
-	prefix  = fmt.Sprintf("%s/%s", Namespace, version)
+	prefix  = fmt.Sprintf("%s/", Namespace)
```

Comment:
I'm leaning toward adding some more identifying info to the prefix so we can implement #25. Maybe something simple like `node.alpha.intel.com/nfd-` as the prefix would help us know what we can safely delete. It's probably not reasonable for NFD to assume it owns the whole `node.alpha.intel.com` label namespace. Related to #27.